shell/platform/darwin/ios/framework/Source/FlutterViewControllerTest.m

```diff
@@ -441,6 +441,7 @@ - (void)testWillDeallocNotification {
   XCTestExpectation* expectation =
       [[XCTestExpectation alloc] initWithDescription:@"notification called"];
   id engine = [[MockEngine alloc] init];
+  @autoreleasepool {
```

Comment:
I tried to reproduce the LUCI error locally, were you able to reproduce it?

Reply:
Yes, you need to run all unittests at once on iOS 12.2 simulator, does not happen with 13.0 or if you run just the single test